src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/model/response/OffenderBookingResponse.kt

```diff
@@ -0,0 +1,3 @@
+package uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.model.response
+
+data class OffenderBookingResponse(val bookingId: Long, val offenderNo: String)
```

Comment:
not sure this needs to be a separate file, but be better in the file where it is referenced instead